modules/openideal_rest/openideal_rest.info.yml

```diff
@@ -0,0 +1,10 @@
+name: 'Openideal Rest'
```

Comment:
Just curious: Why `Openideal Rest` instead of `Openideal Webhooks`?

Reply:
Renamed